cogs/lastactive.py

```diff
@@ -0,0 +1,112 @@
+"""
+lastactive (Ren v2) - Tracks when a user was last active.
+
+How to use:
+    bot.last_active[server_id][channel_id][author_id] -> Retrieves datetime (UTC) of last message sent by that author in
+    that channel.
+    You can change the parameters of the object in the setup method.
+
+Last updated by jangarong on August 21st, 2019.
+"""
+import asyncio
+import pickle
```

Comment:
Curious, is there a specific reason for choosing `pickle` as opposed to using something like `json`?

Reply:
I see that you're serializing the objects themselves, but I think it would be better to just store the guild (server) ID, server ID, and message ID.  This way, if we need to debug, we can also check against the stored file.

Reply:
I still think this type of storage is what databases are for, but JSON is the standard so should probably stick with that